indico/migrations/versions/20171124_1138_2af245be72a6_review_questions_models.py

```diff
@@ -19,6 +19,7 @@
 def upgrade():
     for schema, a, b in (('event_abstracts', 'abstract_review_ratings', 'abstract_review_questions'),
```

Comment:
`a` and `b`, really? ;)

how about some meaningful names. You can always define the list in a separate statement above to keep it readable.